AdminUi/src/AdminUi.Infrastructure.Database.SqlServer/Migrations/20231117101915_RelationshipsOverview.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Backbone.AdminUi.Infrastructure.Database.SqlServer.Migrations
+{
+    /// <inheritdoc />
+    public partial class RelationshipsOverview : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("""
+                CREATE VIEW "RelationshipOverviews" AS
+                    SELECT
+                        "Relationships"."From" AS "From",
+                        "Relationships"."To" AS "To",
+                        "Relationships"."RelationshipTemplateId" AS "RelationshipTemplateId",
+                        "Relationships"."Status" AS "Status",
+                        "Relationships"."CreatedAt" AS "CreatedAt",
+                        "RelationshipChanges"."Res_CreatedAt" AS "AnsweredAt",
+                        "RelationshipChanges"."Req_CreatedByDevice" AS "CreatedByDevice",
+                        "RelationshipChanges"."Res_CreatedByDevice" AS "AnsweredByDevice"
+                    FROM
+                        "Relationships"."Relationships" AS "Relationships"
+                    LEFT JOIN
+                        "Relationships"."RelationshipChanges" AS "RelationshipChanges"
+                    ON
+                        "Relationships"."Id" = "RelationshipChanges"."RelationshipId"
+                    WHERE
+                        "RelationshipChanges"."Type" = 10
```

Comment:
Is this correct SQL Server syntax? I'm under the impression that SQL Server needs brackets (`[` and `]`) instead of double quotes.

Reply:
I just tried resetting the database and tested the migrations and they ran without any problems on my end.
Here is also a screenshot of the query with double quotes in the management studio:
![image](https://github.com/nmshd/backbone/assets/118175875/23f8951e-212f-4862-ad64-005ad99be51f)
So in terms of functionality it seems to be working, but I can change it to keep consistency with the previous migrations.

Reply:
Sorry, I just realized that my "thumb up" wasn't clear: you should change it for consistency reasons, I guess.